packages/@ourworldindata/grapher/src/mapCharts/MapChart.tsx

```diff
@@ -725,6 +785,45 @@ class ChoroplethMap extends React.Component<{
         )
     }
 
+    @computed get globeWidth(): number {
+        return (this.bounds.width - 300) / 2
+    }
+
+    @computed get globeHeight(): number {
+        return (this.bounds.height - 150) / 2
+    }
+
+    @computed get globeScale(): number {
+        if (this.bounds.height < this.bounds.width) {
+            return this.bounds.height / 2.3
+        } else {
+            return this.bounds.width / 2.3
+        }
+    }
+
+    @computed get globeProjection(): any {
+        const worldView = this.manager.projection === MapProjectionName.World
+
+        const scale = worldView ? this.globeScale : this.globeScale * 1.8
```

Comment:
A few [magic numbers](https://stackoverflow.com/a/47902) here? Better as labelled consts? 

The `1.8` and `2.3` getting used above - I think this is probably more important when the contributor isn't a core team member, though I'd flag it in anyone's code